src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -139,6 +140,27 @@ public void visit(AnnotationNode annotation) {
     newline();
   }
 
+  @Override
+  public void visit(NewObjectExpr newObjectValue) {
+    buffer.append(NEW);
+    space();
+    newObjectValue.type().accept(this);
+    // If isGeneric() is true, but generic list is empty, we will append `<>` to the buffer.
+    if (newObjectValue.isGeneric() && newObjectValue.type().reference().generics().isEmpty()) {
+      buffer.append(LEFT_ANGLE).append(RIGHT_ANGLE);
+    }
+    buffer.append(LEFT_PAREN);
+    int size = newObjectValue.arguments().size();
```

Comment:
Nit: `numArguments` instead of `size` which is a rather generic name.

Reply:
got it, thanks! Done.